### ISSUE TITLE
After Changing the GitHub Username, the Gitter Account can be hijacked

### ISSUE THREAD

**Issue by pree** [gitterHQ/gitter#1670]
After changing my GitHub Username, I was able to create a new Account with the old Username and login to gitter with that and gain full access of that Account.

**Comment by mydigitalself** [comment]
Can you elaborate on the steps taken on both Gitter and GitHub in order to achieve this?

**Comment by pree** [comment]
Sure:

* Changed my GitHub Username
* Created a new GitHub Account with the old Username 
* Logged into gitter with the new GitHub Account
* Had Full Access to the Account

**Comment by mydigitalself** [comment]
So @pree already existed on GitHub before and had a Gitter account. Should I delete it so you can recreate and will look into that use case.

**Comment by pree** [comment]
No, I was previously @Schizopriest but changed my Username and it created a new gitter account. I then created a second Github account with that username and this one has access to my old gitter account. 
If you could delete the current gitter account @pree and move the old one (@Schizopriest) to @pree it would be awesome

**Comment by mydigitalself** [comment]
Right, that's a bit more complicated, I'll see what I can do
